Fix: Update the document deploys to current

Rachel-Eichenberger opened the pull request:
Tested this locally, and verified that the deployment URL that is currently in use everywhere still works while this is paginated.

This update changes all the addresses in here to the Newest deployments in use.

 - Added the v2 Routers for LQ
 - All our current networks, expanded no longer mia
 - All known testnets.
 - All WETH addresses
 - Paginated networks
 
<img width="1779" alt="Base_Deployments___Uniswap" src="https://github.com/Uniswap/docs/assets/60412342/de365a21-f755-4134-86d1-0ee3e8b8ed49">

 
<img width="2050" alt="Deployment_Addresses___Uniswap_and_GitHub_Desktop" src="https://github.com/Uniswap/docs/assets/60412342/e3a3251f-b20b-4ac1-9314-8b6ecf3e8e8f">

Rachel-Eichenberger committed:
Fix: Update the document deploys to current

Tested this locally, and verified that the deployment URL that is currently in use everywhere still works while this is paginated.

This update changes all the addresses in here to the Newest deployments in use.

 - Added the v2 Routers for LQ
 - All our current networks, expanded no longer mia
 - All known testnets.
 - All WETH addresses
 - Paginated networks

just-toby commented on docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md:
```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
-| Base                                                 | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6`    |
-| Optimism                                             | `0x0c3c1c532F1e39EdF36BE9Fe0bE1410313E074Bf`    |
-| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
+| Network                                              | Factroy Contract Address                     | V2Router02 Contract Address                  |
```
```suggestion
| Network                                              | Factory Contract Address                     | V2Router02 Contract Address                  |
```

Rachel-Eichenberger replied:
updated in PR fix.

just-toby commented on docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md:
```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
+| Network                                              | Factroy Contract Address                     | V2Router02 Contract Address                  |
+| ---------------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f` | `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D` |
+| Etherum Sepolia                                      | `0xB7f907f7A9eBC822a80BD25E224be42Ce0A698A0` | `0x425141165d3DE9FEC831896C016617a52363b687` |
```
the v2router02 contract here for Sepolia is not verified on etherscan, should we try to get that verified before publishing this?

just-toby replied:
i'm also not seeing any transactions there, so just double checking that it's correct

jsy1218 replied:
Prior deployed UniswapV2Router02 0x7a250d5630B4cF539739dF2C5dAcb4c659F2488 on Sepolia was verified, but it doesn't have the v2 liquidity support. 0x425141165d3DE9FEC831896C016617a52363b687 was retroactively deployed to Sepolia to support v2 liquidity. Probably the deploy was done in a rush so it was not verified.

I believe this was part of the v2-deploy-everywhere, where UniswapV2Router02 needs to support v2 liquidity as well.

jsy1218 replied:
And I think the reason there's no transaction yet is because the PR hasn't merged yet.https://github.com/Uniswap/sdk-core/pull/131

Rachel-Eichenberger replied:
This was newly launched by Emily https://uniswapteam.slack.com/archives/C05MF1J6GR5/p1709845257901909?thread_ts=1708991083.674689&cid=C05MF1J6GR5
The script errored trying to verify the code. Yes, Blast put us behind on pushing the PR.

just-toby commented on docs/contracts/v3/reference/deployments/AVAX Deployments.md:
can we omit spaces from the filenames? i think the title in the sidebar should come from the `title` field. if not, it's fine

just-toby replied:
if possible we should do for all files below too

just-toby commented on docs/contracts/v3/reference/deployments/Arbitrum Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| Contract                                                                                                                                                     | Arbitrum Addresses                           | Arbitrum Sepolia Addresses                   |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x1F98431c8aD98523631AE4a59f267346ea31F984` | `0x248AB79Bbb9bC29bB72f7Cd42F17e054Fc40188e` |
+| [Multicall](https://arbiscan.io/address/0xadF885960B47eA2CD9B55E6DAc6B42b7Cb2806dB#code)                                                                     | `0xadF885960B47eA2CD9B55E6DAc6B42b7Cb2806dB` | `0x2B718b475e385eD29F56775a66aAB1F5cC6B2A0A` |
```
arbitrum mainnet multicall  is not verified on arbiscan

just-toby commented on docs/contracts/v3/reference/deployments/Arbitrum Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` | `0x2779a0CC1c3e0E44D2542EC3e79e3864Ae93Ef0B` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` | `0x101F443B4d1b059569D643917553c771E1b9663E` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0x5E325eDA8064b456f4781070C0738d849c824258` | `0x4A7b5Da61326A6379179b40d00F57E5bbDC962c2` |
```
sepolia UR isn't verified on arbiscan

just-toby commented on docs/contracts/v3/reference/deployments/Arbitrum Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xbfd8137f7d1516D3ea5cA83523914859ec47F573` | `0x0fd18587734e5C2dcE2dccDcC7DD1EC89ba557d9` |
+| [Quoter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/Quoter.sol)                                                              | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` | ``                                           |
+| [SwapRouter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/SwapRouter.sol)                                                           | `0xE592427A0AEce92De3Edee1F18E0157C05861564` | ``                                           |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | ``                                           |
```
NFTDescriptor unverified on arbi mainnet

just-toby commented on docs/contracts/v3/reference/deployments/Arbitrum Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [Quoter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/Quoter.sol)                                                              | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` | ``                                           |
+| [SwapRouter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/SwapRouter.sol)                                                           | `0xE592427A0AEce92De3Edee1F18E0157C05861564` | ``                                           |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | ``                                           |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x91ae842A5Ffd8d12023116943e72A606179294f3` | ``                                           |
```
NonfungibleTokenPositionDescriptor unverified on arbi mainnet

just-toby commented on docs/contracts/v3/reference/deployments/BNB Binance Deployments.md:
```diff
@@ -0,0 +1,58 @@
+| [Multicall](https://bscscan.com/address/0x963Df249eD09c358A4819E39d9Cd5736c3087184#code)                                                                     | `0x963Df249eD09c358A4819E39d9Cd5736c3087184` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0xC9A7f5b73E853664044ab31936D0E6583d8b1c79` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xD9270014D396281579760619CCf4c3af0501A47C` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x831d93E55AF23A2977E4DA892d5005f4F2995071` |
```
unverified

just-toby commented on docs/contracts/v3/reference/deployments/BNB Binance Deployments.md:
```diff
@@ -0,0 +1,58 @@
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0xC9A7f5b73E853664044ab31936D0E6583d8b1c79` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xD9270014D396281579760619CCf4c3af0501A47C` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x831d93E55AF23A2977E4DA892d5005f4F2995071` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x0281E98322e4e8E53491D576Ee6A2BFCE644C55C` |
```
unverified

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x33128a8fC17869897dcE68Ed026d694621f6FDfD` | `0x4752ba5DBc23f44D87826276BF6Fd6b1C372aD24` |
+| [Multicall](https://basescan.org/address/0x091e99cb1c49331a94dd62755d168e941abd0693#code)                                                                    | `0x091e99cb1C49331a94dD62755D168E941AbD0693` | `0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x3334d83e224aF5ef9C2E7DDA7c7C98Efd9621fA9` | `0xD7303474Baca835743B54D73799688990f24a79D` |
```
`0x3334d83e224aF5ef9C2E7DDA7c7C98Efd9621fA9 ` is unverified on basescan

just-toby replied:
same with `0xD7303474Baca835743B54D73799688990f24a79D`

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x0CdeE061c75D43c82520eD998C23ac2991c9ac6d` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xF9d1077fd35670d4ACbD27af82652a8d84577d9F` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x4f225937EDc33EFD6109c4ceF7b560B2D6401009` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0x4615C383F85D0a2BbED973d83ccecf5CB7121463` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` |
```
`0x4615C383F85D0a2BbED973d83ccecf5CB7121463` is unverified on basescan

just-toby replied:
`0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` is unverified on sepolia.basescan.org

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| Contract                                                                                                                                                     |  Base Address                                | Base Sepolia Address                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x33128a8fC17869897dcE68Ed026d694621f6FDfD` | `0x4752ba5DBc23f44D87826276BF6Fd6b1C372aD24` |
+| [Multicall](https://basescan.org/address/0x091e99cb1c49331a94dd62755d168e941abd0693#code)                                                                    | `0x091e99cb1C49331a94dD62755D168E941AbD0693` | `0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1` |
```
i'm seeing `PancakeInterfaceMulticall` contract on [sepolia basecan](https://sepolia.basescan.org/address/0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1#code) here

Rachel-Eichenberger replied:
Yeah this one really confused me, but it was in the docs that were previously updated https://github.com/Uniswap/docs/commit/253006a17c06b47b1ee29df20264c0ea6f157933
The base Sepolia is not in the SDK addresses. I did want to ask Austin about this. It did look the same as ours, and Although the structure is the same, the function name is different so likely a different 4byte function code.

FIgure it's worth keeping for now.. This is just a data passer and should be good to use, the multi-call is extremely helpful to save calls to RPC providers. I will speak with Austin about it.

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [Multicall](https://basescan.org/address/0x091e99cb1c49331a94dd62755d168e941abd0693#code)                                                                    | `0x091e99cb1C49331a94dD62755D168E941AbD0693` | `0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x3334d83e224aF5ef9C2E7DDA7c7C98Efd9621fA9` | `0xD7303474Baca835743B54D73799688990f24a79D` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x0CdeE061c75D43c82520eD998C23ac2991c9ac6d` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xF9d1077fd35670d4ACbD27af82652a8d84577d9F` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
```
`0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` is unverified on sepolia.basescan.org

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x3334d83e224aF5ef9C2E7DDA7c7C98Efd9621fA9` | `0xD7303474Baca835743B54D73799688990f24a79D` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x0CdeE061c75D43c82520eD998C23ac2991c9ac6d` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xF9d1077fd35670d4ACbD27af82652a8d84577d9F` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x4f225937EDc33EFD6109c4ceF7b560B2D6401009` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
```
`0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` is unverified on sepolia.basescan.org

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [V3Migrator](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/V3Migrator.sol)                                                           | `0x23cF10b1ee3AdfCA73B0eF17C07F7577e7ACd2d7` | `0xCbf8b7f80800bd4888Fbc7bf1713B80FE4E23E10` |
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x3d4e44Eb1374240CE5F1B871ab261CD16335B76a` | `0xC5290058841028F1614F3A6F0F5816cAd0df5E27` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x2626664c2603336E57B271c5C0b26F421741e481` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
```
`0x000000000022D473030F116dDEE9F6B43aC78BA3` is unverified on sepolia.basescan

just-toby commented on docs/contracts/v3/reference/deployments/Base Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x3d4e44Eb1374240CE5F1B871ab261CD16335B76a` | `0xC5290058841028F1614F3A6F0F5816cAd0df5E27` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x2626664c2603336E57B271c5C0b26F421741e481` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD` | `0x050E797f3625EC8785265e1d9BDd4799b97528A1` |
```
`0x050E797f3625EC8785265e1d9BDd4799b97528A1` is unverified

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | Optimism Addresses                           | OP Sepolia Addresses  SDK-Core               | OP Sepolia Addresses                         |
```
we discussed in slack but i think maybe we should only have one set of contracts for OP sepolia in the docs

jsy1218 replied:
I just realized that @aadams deployed to **OP Sepolia Addresses** (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204854471729?thread_ts=1706204849.006929&cid=C03BSGBKWA0), meanwhile @sbvegan deployed to **OP Sepolia Addressed SDK-Core** (see https://github.com/Uniswap/deploy-v3/issues/20).

But... I think the corresponding universal router deploy (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204888464989?thread_ts=1706204849.006929&cid=C03BSGBKWA0) happened right before v2-deploy-everywhere, so likely on OP Sepolia, we don't have v2 routing support yet.

I'm a bit leaning toward calling  **OP Sepolia Addresses** (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204854471729?thread_ts=1706204849.006929&cid=C03BSGBKWA0) a canonical set of deployments, and update the SDK, but  would also respect the process, if foundation gets to vote for deploy addresses on testnets as well.

Rachel-Eichenberger replied:
I would not mind shifting to the ones Austin launched. I'm pretty sure there is no v2 testnets outside Goerli fully functional or in the UI (except bad Sepolia), and we need to finish the changes for Sepolia. On the other chains it is still only v3 Sepolia. The [PR mentioned here](https://github.com/Uniswap/docs/pull/683#discussion_r1540145750) was to finish the v2 sepolia testnet.

Maybe we can make a big push to update for v2 and v3 testnets (currently only v3 Goerli rn) and switch to Austin's version. I can leave that version instead here. (it is already in the docs). 

ETH Sepolia should be good for v2 https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/sepolia.json routing, just need the v2router02 [updated in SDK](https://github.com/Uniswap/sdk-core/pull/131)

Making updates for other changes will keep both for now. I think having both in there will let devs know if/why they have had issues. We can rename the column to SDK changing to these OP Sepolia contracts. We will have to update if/when we launch v2 testnets everywhere.
We can speak more on pros vs cons before publishing

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| Contract                                                                                                                                                     | Optimism Addresses                           | OP Sepolia Addresses  SDK-Core               | OP Sepolia Addresses                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x1F98431c8aD98523631AE4a59f267346ea31F984` | `0x8CE191193D15ea94e11d327b4c7ad8bbE520f6aF` | `0x4752ba5DBc23f44D87826276BF6Fd6b1C372aD24` |
+| [Multicall](https://optimistic.etherscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code)                                                         | `0x1F98415757620B543A52E61c46B32eB19261F984` | `0x80e4e06841bb76AA9735E0448cB8d003C0EF009a` | `0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1` |
```
OP mainnet multicall has the name `AlgebraInterfaceMulticall` here: https://optimistic.etherscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code

Rachel-Eichenberger replied:
Yeah, I believe that had to do with the way OP worked their upgrade, It did come out with the same address as the mainnet/default contracts, but when they updated the OP blockchain they relaunched from a state, and all these had to be fixed iirc.

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x91ae842A5Ffd8d12023116943e72A606179294f3` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0xEe6A57eC80ea46401049E92587E52f5Ec1c24785` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` |
+| [NonfungiblePositionManager](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungiblePositionManager.sol)                           | `0xC36442b4a4522E871399CD717aBDD847Ab11FE88` | `0xdA75cEf1C93078e8b736FCA5D5a30adb97C8957d` | `0x27F971cb582BF9E50F397e4d29a5C7A34f11faA2` |
+| [V3Migrator](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/V3Migrator.sol)                                                           | `0xA5644E29708357803b5A882D272c41cC0dF92B34` | `0xE7EcbAAaA54D007A00dbb6c1d2f150066D69dA07` | `0xCbf8b7f80800bd4888Fbc7bf1713B80FE4E23E10` |
```
for some reason i'm not seeing a contract at this address (OP mainnet): https://optimistic.etherscan.io/address/0xA5644E29708357803b5A882D272c41cC0dF92B34

although it has the correct "Public Name Tag"

Rachel-Eichenberger replied:
Removed for now, until know if can be fixed (this was there before OP upgrade, must not have been handled like the rest of the contracts since there wasn't a v2, will need to be removed from SDK and UI if not able to be fixed. I asked in slack, waiting on response.

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x1F98431c8aD98523631AE4a59f267346ea31F984` | `0x8CE191193D15ea94e11d327b4c7ad8bbE520f6aF` | `0x4752ba5DBc23f44D87826276BF6Fd6b1C372aD24` |
+| [Multicall](https://optimistic.etherscan.io/address/0x1F98415757620B543A52E61c46B32eB19261F984#code)                                                         | `0x1F98415757620B543A52E61c46B32eB19261F984` | `0x80e4e06841bb76AA9735E0448cB8d003C0EF009a` | `0xd867e273eAbD6c853fCd0Ca0bFB6a3aE6491d2C1` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2` | `0xD7303474Baca835743B54D73799688990f24a79D` | `0xD7303474Baca835743B54D73799688990f24a79D` |
```
the sdk-core address (0xD7303474Baca835743B54D73799688990f24a79D) is not verified here for sepolia

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xbfd8137f7d1516D3ea5cA83523914859ec47F573` | `0xCb7f54747F58F8944973cea5b8f4ac2209BadDC5` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
+| [Quoter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/Quoter.sol)                                                              | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` | ``                                           | ``                                           |
+| [SwapRouter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/SwapRouter.sol)                                                           | `0xE592427A0AEce92De3Edee1F18E0157C05861564` | ``                                           | ``                                           |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
```
`0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` is not verified on sepolia explorer

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [Quoter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/Quoter.sol)                                                              | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` | ``                                           | ``                                           |
+| [SwapRouter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/SwapRouter.sol)                                                           | `0xE592427A0AEce92De3Edee1F18E0157C05861564` | ``                                           | ``                                           |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x91ae842A5Ffd8d12023116943e72A606179294f3` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
```
`0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` not verified on sepolia explorer

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [SwapRouter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/SwapRouter.sol)                                                           | `0xE592427A0AEce92De3Edee1F18E0157C05861564` | ``                                           | ``                                           |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x91ae842A5Ffd8d12023116943e72A606179294f3` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0xEe6A57eC80ea46401049E92587E52f5Ec1c24785` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` |
```
`0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` not verified for sepolia

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x42B24A95702b9986e82d421cC3568932790A48Ec` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` | `0x4e0caFF1Df1cCd7CF782FDdeD77f020699B57f1a` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x91ae842A5Ffd8d12023116943e72A606179294f3` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` | `0xd7c6e867591608D32Fe476d0DbDc95d0cf584c8F` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0xEe6A57eC80ea46401049E92587E52f5Ec1c24785` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` | `0x1E2A708040Eb6Ed08893E27E35D399e8E8e7857E` |
+| [NonfungiblePositionManager](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungiblePositionManager.sol)                           | `0xC36442b4a4522E871399CD717aBDD847Ab11FE88` | `0xdA75cEf1C93078e8b736FCA5D5a30adb97C8957d` | `0x27F971cb582BF9E50F397e4d29a5C7A34f11faA2` |
```
`0xdA75cEf1C93078e8b736FCA5D5a30adb97C8957d` not verified for sepolia

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [NonfungiblePositionManager](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungiblePositionManager.sol)                           | `0xC36442b4a4522E871399CD717aBDD847Ab11FE88` | `0xdA75cEf1C93078e8b736FCA5D5a30adb97C8957d` | `0x27F971cb582BF9E50F397e4d29a5C7A34f11faA2` |
+| [V3Migrator](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/V3Migrator.sol)                                                           | `0xA5644E29708357803b5A882D272c41cC0dF92B34` | `0xE7EcbAAaA54D007A00dbb6c1d2f150066D69dA07` | `0xCbf8b7f80800bd4888Fbc7bf1713B80FE4E23E10` |
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` | `0x0FBEa6cf957d95ee9313490050F6A0DA68039404` | `0xC5290058841028F1614F3A6F0F5816cAd0df5E27` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` |
```
`0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` not verified for sepolia

just-toby commented on docs/contracts/v3/reference/deployments/Optimism Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` | `0x0FBEa6cf957d95ee9313490050F6A0DA68039404` | `0xC5290058841028F1614F3A6F0F5816cAd0df5E27` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` | `0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0xCb1355ff08Ab38bBCE60111F1bb2B784bE25D7e8` | `0xD5bBa708b39537d33F2812E5Ea032622456F1A95` | `0xD5bBa708b39537d33F2812E5Ea032622456F1A95` |
```
`0xD5bBa708b39537d33F2812E5Ea032622456F1A95` not verified on sepolia

just-toby commented on docs/contracts/v3/reference/deployments/Polygon Deployments.md:
```diff
@@ -0,0 +1,61 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0xec7BE89e9d109e7e3Fec59c222CF297125FEFda2` | `0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD` |
```
UR isn't verified on mumbai: https://mumbai.polygonscan.com/address/0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD#code

just-toby reviewed (approved):
the page structure changes LGTM.
I sanity checked each address on the relevant explorers, and commented for each one that couldn't be validated that way (mostly because contracts aren't verified).

jsy1218 commented on docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md:
```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
+| BNB Chain                                            | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| Base                                                 | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| Optimism                                             | `0x0c3c1c532F1e39EdF36BE9Fe0bE1410313E074Bf` | `0x4A7b5Da61326A6379179b40d00F57E5bbDC962c2` |
+| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
```
I think we should also add blast.

Rachel-Eichenberger replied:
I was planning on waiting till live in the interface, but happy to give devs a running head start... I do need to verify that the contracts are finalized, but I have it ready to go. If I can get verification soon I can add or just go with this and add when verified they are good to go

jsy1218 commented on docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md:
```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
-| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
+| Network                                              | Factroy Contract Address                     | V2Router02 Contract Address                  |
+| ---------------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f` | `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D` |
```
Do we still need to list the deprecated testnets?

Rachel-Eichenberger replied:
Good point. I'll remove all but Goerli, which still existing for now and only available v2

jsy1218 replied:
Hmmm.. Goerli validators stopped producing blocks for a while https://goerli.etherscan.io/, although that precedes EF's planned deprecation date

jsy1218 replied:
And I actually had to disable Goerli from routing all together because of that. https://github.com/Uniswap/routing-api/pull/500

Rachel-Eichenberger replied:
Doh, yeah, well we knew this was coming. Will have to push FE to update the supported list.

jsy1218 commented on docs/contracts/v3/reference/deployments/AVAX Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [Multicall](https://snowtrace.io/address/0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2/contract/43114/code)                                                     | `0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x9AdA7D7879214073F40183F3410F2b3f088c6381` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xEB9fFC8bf81b4fFd11fb6A63a6B0f098c6e21950` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | ``                                           |
```
nftDescriptorLibraryAddressV1_3_0 : 0x27Dd7eE7fE723e83Bf3612a75a034951fe299E99

jsy1218 commented on docs/contracts/v3/reference/deployments/AVAX Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x9AdA7D7879214073F40183F3410F2b3f088c6381` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xEB9fFC8bf81b4fFd11fb6A63a6B0f098c6e21950` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | ``                                           |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | ``                                           |
```
nonfungibleTokenPositionDescriptorAddressV1_3_0: 0xe89B7C295d73FCCe88eF263F86e7310925DaEBAF

jsy1218 commented on docs/contracts/v3/reference/deployments/Arbitrum Deployments.md:
```diff
@@ -0,0 +1,62 @@
+| [Multicall](https://arbiscan.io/address/0xadF885960B47eA2CD9B55E6DAc6B42b7Cb2806dB#code)                                                                     | `0xadF885960B47eA2CD9B55E6DAc6B42b7Cb2806dB` | `0x2B718b475e385eD29F56775a66aAB1F5cC6B2A0A` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2` | ``                                           |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xbfd8137f7d1516D3ea5cA83523914859ec47F573` | `0x0fd18587734e5C2dcE2dccDcC7DD1EC89ba557d9` |
+| [Quoter](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/Quoter.sol)                                                              | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` | ``                                           |
```
quoterV2 arb_sepolia: 0x2779a0CC1c3e0E44D2542EC3e79e3864Ae93Ef0B

Rachel-Eichenberger replied:
This one was verified so I'm assuming you were talking about the blank spaces. The blank is the original quoter not launched with the new contract sets. The 0x2779....Ef0B is present in the v2quoter row

jsy1218 replied:
right

jsy1218 reviewed (approved):
Thank you @Rachel-Eichenberger for putting this together. :)

Rachel-Eichenberger committed:
Fix: Clean up commit

Clean up commit::
Changed filenames (Removed Spaces)
removed mention of depreciated testnets in v2
Fix spelling mistake "Factory" in v2 deploys file
Removed missing Optimism v3 migrator
Added two missing AVAX verified by https://github.com/jsy1218

Tested locally again. Hopefully didn't miss anything. (more to come on OPsep)

Rachel-Eichenberger committed:
Fix: Remove Goerli

Removed all references of Goerli based on depreciation and no longer in SDK.

Goerli is being depreciated, and has stopped producing some blocks...
Goerli has been removed from the SDK

https://github.com/Uniswap/docs/pull/683#discussion_r1541939153
https://github.com/Uniswap/routing-api/pull/500

Rachel-Eichenberger committed:
fix: Add Blast Network

Adds the Blast Network just launched into the documents addresses

snreynolds commented on docs/contracts/v3/reference/deployments/Ethereum-Deployments.md:
```diff
@@ -0,0 +1,63 @@
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD` | `0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD` |
+| [v3StakerAddress](https://github.com/Uniswap/v3-staker)                                                                                                      | `0xe34139463bA50bD61336E0c446Bd8C0867c6fE65` | ``                                           |
+
```
One address that could be worth adding is the v1MixedRouteQuoterAddress... we have it documented in our sdk https://github.com/Uniswap/sdk-core/blob/main/src/addresses.ts#L78C31-L78C73

snreynolds replied:
note that its also on goerli but at a different address

Rachel-Eichenberger replied:
Yes, this would be a great, addition. I also had a couple of other thoughts too. I'll do that in another PR, if you don't think it's a blocker here.

snreynolds commented on docs/contracts/v3/reference/deployments/AVAX-Deployments.md:
```diff
@@ -0,0 +1,59 @@
+| [Multicall](https://snowtrace.io/address/0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2/contract/43114/code)                                                     | `0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x9AdA7D7879214073F40183F3410F2b3f088c6381` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xEB9fFC8bf81b4fFd11fb6A63a6B0f098c6e21950` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x27Dd7eE7fE723e83Bf3612a75a034951fe299E99` |
```
interesting - I dont think this one has been verified on AVAX. But I did do some checking internally and it seems like its correct. How did you originally find this address?

snreynolds replied:
same with NonfungibleTokenPositionDescriptor

Rachel-Eichenberger replied:
https://github.com/Uniswap/docs/pull/683#discussion_r1540120185

I think @jsy1218 grabbed the same way I have in the past, from the deployment address

snreynolds commented on docs/contracts/v3/reference/deployments/Blast-Deployments.md:
```diff
@@ -0,0 +1,55 @@
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x2E95185bCdD928a3e984B7e2D6560Ab1b17d7274` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xAa32bD3926097fd04d22b4433e9867417EE79333` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x497089D9450BB58f536c38c1C0d0A37472303508` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0xB22Ef02E13B1900EBF10391e57162402c11BfF05` |
```
also not verified, worth noting so we can try to

Rachel-Eichenberger replied:
Got the names from the Blast deployment team in slack, and verified contracts on the deployer address

snreynolds commented on docs/contracts/v3/reference/deployments/Celo-Deployments.md:
```diff
@@ -0,0 +1,60 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x82825d0554fA07f7FC52Ab63c961F330fdEFa8E8` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x5615CDAb10dc425a742d643d949a7F474C01abc4` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0x643770E279d5D0733F21d6DC03A8efbABf3255B4` |
```
also not verified... but celo i remember usually didnt work for verification 😆

Rachel-Eichenberger replied:
yeah, I was thinking of either making a push to get some of these non-verified ones verified (just-toby also brought up a lot), or making a script and doing it myself, but yeah another one that gave them issues when launching. [verified in SDK here](https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/celo.json)

snreynolds commented on docs/contracts/v3/reference/deployments/Optimism-Deployments.md:
```diff
@@ -0,0 +1,61 @@
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | Optimism Addresses                           | OP Sepolia Addresses  SDK-Core               | OP Sepolia Addresses                         |
```
what are differences in columns op sepolia sdk-core vs op sepolia? Would lean towards just choosing the canonical one and documenting here.

Rachel-Eichenberger replied:
Yes, so that was something I caught doing this. The SDK and the documents website had two different versions. I was talking with jsy1218 and just-toby about moving it to a canonical version, matching the SDK. Since Both are in use by developers, I put both and when the SDKs are updated I'll remove

snreynolds commented on docs/contracts/v3/reference/deployments/Polygon-Deployments.md:
```diff
@@ -0,0 +1,61 @@
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` | `0x61fFE014bA17989E743c5F6cB21bF9697530B21e` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` | `0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0xec7BE89e9d109e7e3Fec59c222CF297125FEFda2` | `0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD` |
```
this addr 0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD is also UR on polygon - is that just an old one?

Rachel-Eichenberger replied:
yes correct that is the old version that didn't get updated in Mumbai for the v2 yet, but polygon mainnet has the new v2 supported UR
